dataframe/dataframe.go

```diff
@@ -1213,6 +1280,8 @@ func dataframeAppend(_ *starlark.Thread, b *starlark.Builtin, args starlark.Tupl
 		return nil, err
 	}
 
+	// TODO(dustmop): append another DataFrame should work
+
```

Comment:
This is outside of the purpose of the PR, but in this context I think it'd be great to detect the type & returns a variant of `methNotImpl`.